### PR TITLE
Upgrade to cyclops 9 / cyclops-react 2

### PR DESCRIPTION
**Pull request by johnmcclean-aol**
Impl for #336

### CODE REVIEW

File: micro-core/src/main/java/com/aol/micro/server/module/ConfigurableModule.java [hunk 1]
```diff
@@ -18,40 +18,57 @@
...
 
 @Builder
 @AllArgsConstructor
-@Wither
+
 public class ConfigurableModule implements Module{
 
+	@Wither
 	private final Set<Object> jaxRsResourceObjects;
+    @Wither
```
Comment (tonyfinn) [inline micro-core/src/main/java/com/aol/micro/server/module/ConfigurableModule.java]: Indentation is messed up here.

File: micro-couchbase/src/main/java/com/aol/micro/server/couchbase/CouchbaseDistributedMapClient.java [hunk 2]
```diff
@@ -3,10 +3,10 @@
 import java.util.Optional;
 import java.util.concurrent.ExecutionException;
 
+import com.aol.cyclops2.util.ExceptionSoftener;
```
Comment (tonyfinn) [inline micro-couchbase/src/main/java/com/aol/micro/server/couchbase/CouchbaseDistributedMapClient.java]: Why is it still com.aol.cyclops(2) here while it's cyclops.X elsewhere. Is this API not intended for use outside cyclops?

Comment (johnmcclean-aol) [inline micro-couchbase/src/main/java/com/aol/micro/server/couchbase/CouchbaseDistributedMapClient.java]: It's non-core, cyclops is very big. The core data types are defined in cyclops.X, everything else stays as com.aol.cyclops, and internal stuff is at com.aol.cyclops.internal.

File: micro-core/src/main/java/com/aol/micro/server/module/ConfigurableModule.java (hunk 1 above)
Comment (johnmcclean-aol) [inline micro-core/src/main/java/com/aol/micro/server/module/ConfigurableModule.java]: It's strange, I'm not sure why. Looks fine in IntelliJ - should be spaces not tabs too. Will play around a bit more.

**Pull request merged**